### PR TITLE
dev-java/openjdk: update riscv patch for 17.0.5, #878441

### PR DESCRIPTION
**Pull request by YangMame**
tatt build report:
USE tests started on Fri Nov 18 01:25:19 CST 2022                      
                                                                               
FEATURES=' test' USE='' succeeded for =dev-java/openjdk-17.0.5_p8
USE='alsa cups -doc examples -headless-awt jbootstrap source -system-bootstrap -systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='-alsa -cups doc examples headless-awt jbootstrap source -system-bootstrap -systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='-alsa cups doc examples headless-awt jbootstrap source -system-bootstrap -systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='-alsa -cups doc examples -headless-awt -jbootstrap -source system-bootstrap -systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='-alsa cups -doc -examples headless-awt -jbootstrap -source system-bootstrap -systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='alsa -cups doc -examples -headless-awt jbootstrap -source system-bootstrap -systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='alsa -cups doc -examples headless-awt jbootstrap -source system-bootstrap -systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='alsa -cups doc examples headless-awt -jbootstrap source system-bootstrap -systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='alsa cups -doc -examples -headless-awt jbootstrap source system-bootstrap -systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='alsa -cups -doc -examples headless-awt jbootstrap source -system-bootstrap systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='alsa cups doc examples headless-awt jbootstrap source -system-bootstrap systemtap' succeeded for =dev-java/openjdk-17.0.5_p8
USE='-alsa -cups -doc -examples headless-awt -jbootstrap source system-bootstrap systemtap' succeeded for =dev-java/openjdk-17.0.5_p8

### CODE REVIEW

**Comment by YangMame** [comment]
maybe need someone upload this patch to dev distfiles

File: dev-java/openjdk/openjdk-17.0.5_p8.ebuild [hunk 1]
```diff
@@ -46,11 +46,11 @@ SRC_URI="
 		$(bootstrap_uri x86 ${X86_XPAK})
 		$(bootstrap_uri riscv ${RISCV_XPAK})
 	)
-	riscv? ( https://dev.gentoo.org/~arthurzam/distfiles/dev-java/openjdk/openjdk-17.0.3-riscv.patch.xz )
+	riscv? ( https://github.com/felixonmars/archriscv-packages/raw/master/java17-openjdk/java17-riscv64.patch )
```
Comment (Flowdalic) [inline dev-java/openjdk/openjdk-17.0.5_p8.ebuild:49]: Please use a stable URL for the patch

```suggestion
	riscv? ( https://raw.githubusercontent.com/felixonmars/archriscv-packages/24658ef25980d72c7493d99c35922ab24f4a7799/java17-openjdk/java17-riscv64.patch )
```

**Comment by Flowdalic** [comment]
I would also recommend to squash the commits into one.

**Comment by gyakovlev** [comment]
thanks!
I re-uploaded the file to my devspace and updated ebuild accordingly.

**Pull request closed without merging**